oaknational/Oak-Web-Application
# Feat/lesq 1025/sign up theming [LESQ-1025]

## Description

Music year: 2004

- put sign up and sign in pages into a group and create a layout
- apply styling and theming to clerk components 
- the oak-components library does not work out of the box with app router, so have re-exported the whole lib with the 'use client' flag for usage in these pages
- add a providers.ts to re-export posthog provider for use in app pages


## How to test

1. Go to https://deploy-preview-2711--oak-web-application.netlify.thenational.academy/sign-up (while signed out of the app)
2. Click on 'already have an account? sign in' to go to sign in page
3. You should see the pages look nice ✨ 

## Screenshots

How it should now look:
<img width="700" alt="Screenshot 2024-08-19 at 10 54 59" src="https://github.com/user-attachments/assets/c7530815-61af-4ec0-8045-eba2423bc910">

<img width="200" alt="Screenshot 2024-08-16 at 14 56 57" src="https://github.com/user-attachments/assets/a48cfcdb-cd48-47cd-b939-b45e5e92bd64">


## Checklist
- [ ]  sign-up and sign-in pages should have an oak image backround on desktop
- [ ]  the pages should have a white background on mobile
- [ ]  the clerk sign-up/sign-in components should be themed (oak font and colours)
- [ ]  the pages should be inaccesible without the feature flag
